notebooks/official/custom/custom_training_tensorboard_profiler.ipynb

```diff
@@ -0,0 +1,1631 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    if os.getenv("IS_TESTING_TF"):
</pre><p>What is this envvar? Remove if possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
Removed the env variable.